src/app/filtering-block/filtering-block.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="filtering-block">
+    <div class="sort">
+        <span class="sort-block">Sorting by:</span>
+        <button class="sort-btn" (click)="onDateChange()">date</button><span class="sort-span" *ngIf="dateTermF === true"> &uarr;</span><span class="sort-span" *ngIf="dateTermF === false"> &darr;</span>
+        <button class="sort-btn" (click)="onViewsChange()">count of views </button><span class="sort-span" *ngIf="viewsTermF === true"> &uarr;</span><span class="sort-span" *ngIf="viewsTermF === false"> &darr;</span>
+    </div>
+    <div><label class="filter-label"> by word or sentance<input class="filter-input" #box type="text" (keyup)="onFilterChange(box.value)"></label></div>
```

Comment:
```suggestion
    <div><label class="filter-label"> by word or sentence<input class="filter-input" #box type="text" (keyup)="onFilterChange(box.value)"></label></div>
```

if you use VSCode I suggest you install CodeSpell Checker it helps you avoid misspellings

Reply:
I fixed it, thank you